lab_2_tokenize_by_bpe/main.py

```diff
@@ -69,6 +137,21 @@ def get_vocabulary(
     :param unknown_token: a token to signify an unknown token
     :return: dictionary in the form of <token: identifier>
     """
+    if (not isinstance(word_frequencies, dict)
+            or not isinstance(unknown_token, str)):
+        return None
+    tokens = [unknown_token]
```

Comment:
why not a `set`?

Reply:
now set() helps shrink the list after finding all possible tokens